.azure-pipelines/warden.yml

```diff
@@ -24,6 +24,34 @@ jobs:
       python setup.py sdist -d $(Build.ArtifactStagingDirectory) --format zip
     displayName: 'Build Package'
 
+  - task: CopyFiles@2
+    displayName: 'Copy files for github release'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)'
+      Contents: 'scripts/*/verify_agent_os.py'
+      TargetFolder: '$(Build.BinariesDirectory)/sdk-tools'
+      OverWrite: true
+      flattenFolders: true
+  
+  - task: ArchiveFiles@2
```

Comment:
Do you know if the explicit archiving required? or if the GitHubRelease task will do that as part of the upload.

Reply:
Yes explicitly archiving is required, else the release task will output individual files.